tests/v2/validation/rbac/psa/psa_test.go

```diff
@@ -115,27 +117,27 @@ func (rb *PSATestSuite) ValidatePSA(role string, customRole bool) {
 
 	rb.T().Logf("Validate deletion of the PSA labels as %v", role)
 
-	deleteLabels(labels)
+	deletePSALabels(labels)
 
 	deleteLabelsNS, err := getAndConvertNamespace(rb.adminNamespace, rb.steveAdminClient)
 	require.NoError(rb.T(), err)
 	deleteLabelsNS.Labels = labels
 
 	_, err = rb.steveNonAdminClient.SteveType(namespaces.NamespaceSteveType).Update(rb.adminNamespace, deleteLabelsNS)
 	switch role {
-	case restrictedAdmin, roleOwner:
+	case rbac.RestrictedAdmin, rbac.RoleClusterOwner:
 		require.NoError(rb.T(), err)
 		expectedLabels := getPSALabels(response, labels)
 		assert.Equal(rb.T(), 0, len(expectedLabels))
 		if !customRole {
 			_, err = createDeploymentAndWait(rb.steveNonAdminClient, rb.client, rb.cluster.ID, containerName, containerImage, rb.adminNamespace.Name)
 			require.NoError(rb.T(), err)
 		}
-	case roleMember, roleProjectReadOnly:
+	case rbac.RoleClusterMember, rbac.RoleReadOnly:
 		require.Error(rb.T(), err)
 		errMessage := strings.Split(err.Error(), ":")[0]
 		assert.Equal(rb.T(), "Resource type [namespace] is not updatable", errMessage)
-	case roleProjectOwner, roleProjectMember, roleCustomCreateNS:
+	case rbac.RoleProjectOwner, rbac.RoleProjectMember, rbac.RoleCustomCreateNS:
 		errStatus := strings.Split(err.Error(), ".")[1]
```

Comment:
should there be a require.Error here?

Reply:
yes, we have it on line 137.

Reply:
Looks like require.Error is still missing?